pkg/volume/azure_file/azure_file.go

```diff
@@ -114,6 +114,9 @@ func (plugin *azureFilePlugin) newMounterInternal(spec *volume.Spec, pod *v1.Pod
 		return nil, err
 	}
 	secretName, secretNamespace, err := getSecretNameAndNamespace(spec, pod.Namespace)
+	if err != nil {
```

Comment:
Just to confirm, do we intentionally want to change the behavior here? In other words, are we comfortable that an `err` in `getSecretNameAndNamespace` will now cause `newMounterInternal` to return an error?

It could be worth proposing this change in a separate pr :)

Reply:
@mattjmcnaughton seeing azure e2e tests failing, I think I'll go with the safer fix for now, and will just log the error here, and in other places, w/o them being returned. Can you suggest what level of log you think is appropriate here

Reply:
I defer to the Azure folks on what's best :) I don't have a ton of context. But I definitely agree that's less risky :)

Reply:
@mattjmcnaughton having error logged as a part of this pr wouldn't hurt, and could useful for debugging at least :) Once it's merged, I can open a separate pr with the error returned, as you've suggested earlier + will tag Azure folks there.